lib/modules/versioning/ubuntu/index.spec.ts

```diff
@@ -269,43 +272,50 @@ describe('modules/versioning/ubuntu/index', () => {
   );
 
   it.each`
-    a           | b            | expected
-    ${'20.04'}  | ${'2020.04'} | ${false}
-    ${'17.10'}  | ${'artful'}  | ${true}
-    ${'xenial'} | ${'artful'}  | ${false}
-    ${'17.04'}  | ${'artful'}  | ${false}
-    ${'artful'} | ${'17.10'}   | ${true}
-    ${'16.04'}  | ${'xenial'}  | ${true}
-    ${'focal'}  | ${'20.04'}   | ${true}
-    ${'20.04'}  | ${'focal'}   | ${true}
-    ${'19.10'}  | ${'19.10'}   | ${true}
+    a                   | b                   | expected
+    ${'20.04'}          | ${'2020.04'}        | ${false}
+    ${'17.10'}          | ${'artful'}         | ${true}
+    ${'xenial'}         | ${'artful'}         | ${false}
+    ${'17.04'}          | ${'artful'}         | ${false}
+    ${'artful'}         | ${'17.10'}          | ${true}
+    ${'16.04'}          | ${'xenial'}         | ${true}
+    ${'focal'}          | ${'20.04'}          | ${true}
+    ${'20.04'}          | ${'focal'}          | ${true}
+    ${'19.10'}          | ${'19.10'}          | ${true}
+    ${'jammy'}          | ${'jammy-20230816'} | ${true}
```

Comment:
This might be ok, but it's a bit strange

Reply:
That's fair. Would you prefer it to return false?

Easy enough to check if both have a container image date and compare the date as well.

Reply:
I think false. Otherwise a==b and a==c but b !== c if b and c are dated